hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapLoaderTest.java

```diff
@@ -72,12 +74,26 @@ public void testSenderAndBackupTerminates_AfterInitialLoad() throws InterruptedE
         HazelcastInstance[] ownerAndReplicas = findOwnerAndReplicas(instances, name);
         ownerAndReplicas[0].getLifecycleService().terminate();
         ownerAndReplicas[1].getLifecycleService().terminate();
+        waitUntilActive(ownerAndReplicas[3], 15);
 
         map = ownerAndReplicas[3].getMap(name);
         map.loadAll(false);
         assertEquals(DummyMapLoader.SIZE, map.size());
     }
 
+    private void waitUntilActive(HazelcastInstance hz, int timeoutSeconds) {
```

Comment:
Should we put this in `HazelcastTestSupport`? It looks useful for other tests as well.